e2e-cypress/files/README.md

```diff
@@ -43,6 +43,34 @@ In order to generate one xml report per test type (installation, integration and
 With Cypress 12 version is now available `cy.origin()` that allows you to handle redirections. This funcionality eases the login handling.
 See `./support/e2e.ts` for a generic login example.
 
+## Cypress Cloud
+
+To use Cypress Cloud, follow these steps:
+1. **Request access.** In order to access Cypress Cloud, you will need to request access to the AAD group. This can be done through MyServices using the Active Directory Group - Administration Request form. Once access has been granted, you will be able to log into https://cloud.cypress.io/login using your mail.
```

Comment:
This point refers to internal processes of a specific organization. I would not mention those within an open source project. Maybe rephrase this and the following point to be more general.

Reply:
This is pending to be changed from their side already.